docs/containers/quickstart-node.md

```diff
@@ -35,16 +35,28 @@ In this guide you will learn how to:
 1. Open the project folder in VS Code.
 1. Open the Command Palette (`kb(workbench.action.showCommands)`) and use **Docker: Add Docker Files to Workspace...** command:
 
-   ![Add Dockerfile to a Node.js project](images/quickstarts/node-add-node.png)
+   ![Add Dockerfile to a Node.js project](images/quickstarts/node-add-node-dark.png)
 
 1. Select **Node.js** when prompted for the application platform.
-1. Select either **Yes** or **No** when prompted to include Docker Compose files.
-
-   > The Docker Compose files are optional and not used for debugging the application within a container, so either is a valid choice.
+1. Select either **Yes** or **No** when prompted to include Docker Compose files. Compose is typically used when running multiple containers at once.
 
 1. Enter `3000` when prompted for the application port.
 
-The extension will create `Dockerfile` and `.dockerignore` files. If you elected to include Docker Compose files, `docker-compose.yml` and `docker-compose.debug.yml` will be generated as well. Finally, the extension will create a set of **VS Code tasks** in `.vscode/tasks.json` for building and running the container (in both debug- and release-configurations) and a **launch debug configuration** in `.vscode/launch.json` for debugging the service within the container.
+The extension creates `Dockerfile` and `.dockerignore` files. If you elected to include Docker Compose files, `docker-compose.yml` and `docker-compose.debug.yml` will be generated as well. Finally, the extension will create a set of **VS Code tasks** in `.vscode/tasks.json` for building and running the container (in both debug- and release-configurations) and a **launch debug configuration** in `.vscode/launch.json` for debugging the service within the container.
+
+## Add an environment variable to the image
+
+The Docker extension helps you author Dockerfiles by using [IntelliSense](../editor/intellisense.md) to provide auto-completions and contextual help. To see this feature in action, add an environment variable to your service image by following these steps:
+
+1. Open the `Dockerfile` file.
+1. Use `ENV` instruction to add an environment variable to the service container image. The instruction should be placed in the `base` stage of the `Dockerfile` (the first stage in the file).
```

Comment:
The part about using the base stage of the Dockerfile applies to .NET, where we start from that stage to run a special version of the container for debugging, file-mapping debugger and application bits into the container.

All this wizardry is not necessary for Node which has a debugger (they call it "inspector") built-in into the runtime. Please double-check, but I do not think our extension scaffolds multi-stage Dockerfile for node. If my memory is right, none of the information referring to the base stage in the Dockerfile would apply to node, i.e. to this quickstart